test/src/sw/redis++/async_test.h

```diff
@@ -199,6 +199,13 @@ void AsyncTest<RedisInstance>::_test_hash() {
                 this->set_ready();
             });
     _wait();
+
+    set_ready(false);
+    std::map<std::string, std::string> mkeys = {{test_key("str")+"1", "val1"}, {test_key("str")+"2", "val2"}};
```

Comment:
Use std::unordered_map instead of std::map, since this file doesn't include <map>, some compiler might fail to compile.